cms/djangoapps/contentstore/apps.py

```diff
@@ -0,0 +1,22 @@
+"""
+Contentstore Application Configuration
+
+Above-modulestore level signal handlers are connected here.
+"""
+
+from django.apps import AppConfig
+
+
+class ContentstoreConfig(AppConfig):
+    """
+    Application Configuration for Grades.
+    """
+    name = u'cms.djangoapps.contentstore'
```

Comment:
I think there's something else trying to register this name and the collision is causing a build breakage.